backlog_controller.hh

```diff
@@ -39,7 +39,7 @@ class backlog_controller {
 public:
     struct scheduling_group {
         seastar::scheduling_group cpu = default_scheduling_group();
-        const ::io_priority_class& io = default_priority_class();
+        seastar::io_priority_class io = default_priority_class();
```

Comment:
At this point the initializers are not needed because their default constructor is equivalent to the `default_*()` factory method result, no?

Reply:
Only the `scheduling_group` has. But `io_priority_class` [is going away](https://github.com/scylladb/seastar/issues/1069) anyway